.github/labeler.yml

```diff
@@ -3,3 +3,14 @@
 dependencies:
   - '**/*requirements*.*'
   - '**/setup.py'
+demo:
+  - any: ['demos/**/*', '!demos/**/*.md']
+documentation:
+  - '*.md'
+  - '**/*.md'
+model:
+  - 'models/**/*.yml'
+accuracy checker:
+  - 'tools/accuracy_checker/**/*'
+downloader:
```

Comment:
we are going to rename folder downloader to model_tools because it is not just about downloader. So, label should be changed later.
https://github.com/openvinotoolkit/open_model_zoo/pull/2723

Reply:
New path is added.